### ISSUE TITLE
tropical-islands is incorrectly packaged

### ISSUE THREAD

**Issue by Moshyfawn** [bensmithett/tropical#35]
### Description
I'm getting this message in my terminal
```cmd
tropical-islands is incorrectly packaged. Please contact the package author to fix.
```
### Steps to reproduce
1. Clone the project generated from [the tropical template](https://github.com/bensmithett/tropical)
2. Run `yarn` cmd
3. Run `yarn dev`
4. Open http://localhost:3000
5. See the message in the terminal
### Details
- OS: MacOS 12.1
- Browser: Firefox 97.0b8 (64-bit)
### Additional information
```cmd
yarn run v1.22.17
$ node server
Pre-bundling dependencies:
  react
  fela
  react-fela
  tropical-islands
  react-dom/server
  (...and 7 more)
(this will be run only when your dependencies or config have changed)
🏝   Your Tropical site is now being served at: http://localhost:3000
tropical-islands is incorrectly packaged. Please contact the package author to fix.
```
_P.S. contacting the package author ;)_

**Comment by bensmithett** [comment]
Thanks @Moshyfawn! I poked at this locally before christmas then forgot about it :) I'll get a fix up asap

**Comment by bensmithett** [comment]
The good news is that this is closed by #38 

The bad news is that my solution was to publish tropical-islands as a [pure ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) package. 

This was something I'd been wanting to do for a while (see #28, and more recently I wanted to switch away from [`vite-plugin-mdx`](https://github.com/brillout/vite-plugin-mdx) to start using MDX 2... but `@mdx-js/rollup` is also pure ESM)

Hopefully upgrading is as easy as just merging the upstream changes from https://github.com/bensmithett/tropical/releases/tag/v10.0.0 but let me know if you have any issues.

**Comment by Moshyfawn** [comment]
Looks good, awesome!

Why would [pure ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) package be the bad news? You mean the other packages compatibility issues? I don't think it's as big of an issue this days.

I'm gonna look into migrating and will report if I get stuck!
